sickbeard/logger.py

```diff
@@ -155,6 +155,19 @@ def submit_errors(self):
         if not (sickbeard.GIT_USERNAME and sickbeard.GIT_PASSWORD and len(classes.ErrorViewer.errors) > 0):
```

Comment:
Should make this line:
`if not all([sickbeard.DEBUG, sickbeard.GIT_USERNAME, sickbeard.GIT_PASSWORD, classes.ErrorViewer.errors]):
(would need to test it to make sure it behaves as expected, if not could just add the sickbeard.DEBUG part in the line above.